### ISSUE TITLE
Unable to find appConfig

### ISSUE THREAD

**Issue by loretoparisi** [Jaffa/amazon-music#25]
For some reasons, there is this error raised by

```python3
if app_config is None:
                raise Exception(
                    "Unable to find appConfig in {}".format(r.content))
```

**Comment by TTSS000** [comment]
Hello.
I'm using Amazon Japan but I hope this applies to all over the world of Amazon.
It seems Amazon changed the format of response body.
So, in amazonmusic.py, I changed some codes.
- step1
Find the line
        if 'amznMusic.appConfig = ' in line:
and change to
        if 'applicationContextConfiguration = ' in line:

- step 2
add the line shown here at just a line below the line modified step1
                    line = re.sub(r'    var applicationContextConfiguration = ', '',line)

Then, I encounter the issue of the second authentication. But it seems I was able to solve it by exporting cookies from my Chrome browser.

I hope this helps.
